src/gui/search/searchwidget.cpp

```diff
@@ -137,8 +137,7 @@ SearchWidget::SearchWidget(MainWindow *mainWindow)
     connect(m_tabStatusChangedMapper, static_cast<void (QSignalMapper::*)(QWidget *)>(&QSignalMapper::mapped)
             , this, &SearchWidget::tabStatusChanged);
 
-    // NOTE: Although SearchManager is Application-wide component now, we still create it the legacy way.
-    auto *searchManager = new SearchPluginManager;
+    auto *searchManager = SearchPluginManager::instance();
```

Comment:
You also need remove the line `delete SearchPluginManager::instance();` from destructor and put it into appropriate place in Application class.

Reply:
I still don't see you removing that code from the destructor.

Reply:
@Piccirello, why are you once again ignoring this comment?

Reply:
Sorry about that, I subsequently missed this comment when reviewing on GitHub.